gnucash/import-export/import-main-matcher.cpp

```diff
@@ -1909,6 +1909,20 @@ get_peer_acct_names (Split *split)
     return retval;
 }
 
+static bool
```

Comment:
`static inline bool`. The compiler will probably inline it anyway, but it's good to put the hint in for humans.